plotnine/geoms/geom_label.py

```diff
@@ -15,6 +15,7 @@
     #
     # Documentation at matplotlib.patches.BoxStyle
     'boxstyle': 'round',
+    'boxcolor': 'black',
```

Comment:
Set the default to `None` so that `boxcolor` only has an effect if the user has set it.

Reply:
I've restored the old behaviour: unset boxcolor means 'use color aes' and documented accordingly.